src/index.js

```diff
@@ -51,10 +56,14 @@ class AutoSSH extends EventEmitter {
 
     this.sshPort = conf.sshPort || 22;
     this.privateKey = conf.privateKey || null;
+    this.useAutossh = conf.useAutossh || false;
+
+    this.executable = (this.useAutossh ? 'autossh' : 'ssh') || '';
+    this.extraOpts = conf.extraOpts || [];
   }
 
   /*
-  */
+     */
```

Comment:
are you using some kind of prettifier to cause all these little changes? If so, which one are you using and I may enforce it as a requirement.

Reply:
That was eslinint --fix i suppose.